docs/commands/push.md

```diff
@@ -1,93 +1,186 @@
+To find your organization ID required for the command:
 
-Note that only API definitions with a CI source can be updated with the `push` command. Attempting to update API definitions created from other sources will fail with an error.
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. In your browser's address bar, find the URL of this page.
+1. Inspect the segment after `app.redoc.ly/org/`. This part is your organization ID.
 
-To create a new API and a new API definition version with the `push` command, use the `-u` option:
+For example, if the URL is `app.redoc.ly/org/test_docs`, the organization ID is `test_docs`. When using the `push` command, you would provide this ID as `@test_docs`.
 
+:::warning Note
+The organization ID can differ from the organization name. Owners can change the organization name at any time in the Workflows **Org settings** page, but the organization ID cannot be changed.
+:::
 
-```bash
-openapi push -u test-api-v1.yaml @redocly/test-api@v1 main
-```
+#### API name
+
+To find your API name required for the command:
+
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. Check the list of APIs displayed on this page.
+1. Inspect the title of each list item to the left of the **New version** and **Edit API** action buttons. This title is an API name.
+
+When using the `push` command, you would provide API name after the [Organization ID](#organization-id) separated with the forward slash (`/`). For example, `@test_docs/petstore-api`.
+
+:::info
+The name of your API definition should contain only supported characters (`a-z`, `A-Z`, `0-9`, `-`, `.`). Using a restricted character will result in an error, and your API definition will not be created.
+:::
 
+#### API version
 
-The name and version of your API definition should contain only supported characters (`a-z`, `A-Z`, `0-9`, `-`, `.`). Using a restricted character will result in an error, and your API definition will not be created.
+To find your API version required for the command:
 
-If the `branchName` option is omitted, the command will use the default branch.
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. Check the list of APIs displayed on this page.
+1. Inspect the sub-title of each list item to the bottom of the [API name](#api-name). This sub-title is an API version.
+
+When using the `push` command, you would provide API version after the [API name](#api-name) separated with the "at" symbol (`@`). For example, `@test_docs/petstore-api@v1`.
+
+:::info
+The version of your API definition should contain only supported characters (`a-z`, `A-Z`, `0-9`, `-`, `.`). Using a restricted character will result in an error, and your API definition will not be created.
+:::
+
+### Run-id
 
 The `--run-id` option can be used by Redocly Workflows to associate multiple pushes with a single CI job. It is auto-populated for the following CI systems so you don't have to pass it separately:
```

Comment:
Do you think it would be useful to investigate and mention a couple of use-cases or reasons why a user would want to do something like this?